server/mailet/mailets/src/test/resources/eml/spam8.eml

```diff
@@ -0,0 +1,321 @@
+From ormlh@imail.ru  Sun Jul 15 04:56:31 2001
```

Comment:
Quick question: where is this mail taken from? Protected by some sort of license?

Can we spend 5 minutes checking there is no issue?

Reply:
> Where?

I cloned it from `third-party/spamassassin/src/test/resources/spamassassin_db/spam/spam8`

Reply:
Ok but had this been decently checked upon that time? My comment is still relevant.

Reply:
checked
It's a unavailable email